src/test/java/seedu/mtracker/console/InputParserTest.java

```diff
@@ -42,49 +47,77 @@ void initialiseInstruments() {
     @Test
     void getIndexNumber_validIndexProvided_expectSuccess() {
         InputParser parser = new InputParser();

```

Comment:
Since this parser is going to be used repeatedly, can declare a private InputParser parser, then in the @beforeEach function, assign parser as new InputParser(), like that for all the methods below can remove the line "InputParser parser = new InputParser();"

^do rename the @BeforeEach function, to like initialiseTestComponents or something